R/fda-table_32.R

```diff
@@ -0,0 +1,69 @@
+# Pre-processing
+
+library(tidyverse)
```

Comment:
We don't need all the `tidyverse` packages, `dplyr` is sufficient. 
```suggestion
library(dplyr)
```

Reply:
Understood.